library/src/main/java/com/feedhenry/sdk/sync/FHSyncConfig.java

```diff
@@ -369,8 +401,220 @@ public static FHSyncConfig fromJSON(JSONObject pObj) {
         return config;
     }
 
+    @Deprecated
     public FHSyncConfig clone() {
         JSONObject json = this.getJSON();
         return FHSyncConfig.fromJSON(json);
     }
+
+    /**
+     * Class for creating FHSyncConfig.
+     */
+    public static class Builder {
```

Comment:
better class name than `Builder` maybe?

Reply:
I think this is ok. Ok, I copy Google's style a little bit. Notification.Builder(), etc.